### ISSUE TITLE
Unable to manipulate attributes

### ISSUE THREAD

**Issue by rtpm** [SaschaKrug/django-component-kit#1]
**Describe the bug**
Unable to manipulate attributes (v 0.5), python 3.12.1

**To Reproduce**
From docs:

```python
@register.tag
@register.tag('endalert')
@component_block_tag(get_template("components/alert.html"))
def alert(attributes: dict) -> dict:
    """Component for rendering an alert."""
    dismissible = attributes.pop("dismissible", False)
    return {
        "dismissible": dismissible,
        "attributes": attributes
    }
```

Getting error:
```
raise TemplateSyntaxError(f"{self.name} component is missing required argument: {key}")
django.template.exceptions.TemplateSyntaxError: alert component is missing required argument: attributes
```

Thanks for this nice library.

**Comment by SaschaKrug** [comment]
Thank you for the support! That's indeed an error in the documentation. Here are the corrected snippets:

```python
# templatetags/mycomponents.py
from django import template
from django.template.loader import get_template
from django_component_kit import component_block_tag

register = template.Library()


@register.tag
@register.tag("endalert")
@component_block_tag(get_template("mycomponents/alert.html"))
def alert(dismissible: bool) -> dict:
    """Component for rendering an alert."""
    return {"dismissible": dismissible}
```

```html
<!-- templates/mycomponents/alert.html -->
{% load django_component_kit %}

<div {% merge_attrs attributes %}>
    {% render_slot slots.children %}
    {% if dismissible %}
    <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
    {% endif %}
</div>
```

No pop or magic `attributes` parameter required. Just provide the attribute you wanna modify as parameters.

Please note: `merge_attrs` is required in the template to actually render the attributes correctly.

I will release a new version with fixed documentation.

**Comment by SaschaKrug** [comment]
Fixed in [0.5.1](https://github.com/SaschaKrug/django-component-kit/releases/tag/0.5.1)

**Comment by rtpm** [comment]
Thank you Sascha!
